<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>비밀번호</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_Edit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border-radius:15px;
border:2px solid #000;
background:#fff;
}

QLineEdit:focus{
border:2px solid #5865f2;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pw_Edit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border-radius:15px;
border:2px solid #000;
background:#fff;
}

QLineEdit:focus{
border:2px solid #5865f2;
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>아이디</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>이메일 주소</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_Edit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border-radius:15px;
border:2px solid #000;
background:#fff;
}

QLineEdit:focus{
border:2px solid #5865f2;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email_Edit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border-radius:15px;
border:2px solid #000;
background:#fff;
}

QLineEdit:focus{
border:2px solid #5865f2;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>이름</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="repw_Edit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border-radius:15px;
border:2px solid #000;
background:#fff;
}

QLineEdit:focus{
border:2px solid #5865f2;
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string> 비밀번호 재입력</string>
   </property>
  </widget>
  <widget class="QPushButton" name="join_Btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:15px;
border:2px solid #6000e4;
background-color:#6000e4;
color:#fff;
}

QPushButton:hover{

}
</string>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pw_Btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:15px;
border:2px solid #6000e4;
background-color:#6000e4;
color:#fff;
}

QPushButton:hover{

}
</string>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_Btn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:15px;
border:2px solid #6000e4;
background-color:#6000e4;
color:#fff;
}

QPushButton:hover{

}
</string>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
  <widget class="QPushButton" name="email_Btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>290</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:15px;
border:2px solid #6000e4;
background-color:#6000e4;
color:#fff;
}

QPushButton:hover{

}
</string>
   </property>
   <property name="text">
    <string>전송</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>인증번호 확인</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailnum_Edit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border-radius:15px;
border:2px solid #000;
background:#fff;
}

QLineEdit:focus{
border:2px solid #5865f2;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="email_C_Btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>350</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:15px;
border:2px solid #6000e4;
background-color:#6000e4;
color:#fff;
}

QPushButton:hover{

}
</string>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>id_Edit</tabstop>
  <tabstop>id_Btn</tabstop>
  <tabstop>pw_Edit</tabstop>
  <tabstop>pw_Btn</tabstop>
  <tabstop>repw_Edit</tabstop>
  <tabstop>name_Edit</tabstop>
  <tabstop>email_Edit</tabstop>
  <tabstop>email_Btn</tabstop>
  <tabstop>emailnum_Edit</tabstop>
  <tabstop>join_Btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
